AlphaWallet/Core/CoinTicker/CoinTickersResolver.swift

```diff
@@ -0,0 +1,243 @@
+//
+//  CoinTickersResolver.swift
+//  AlphaWallet
+//
+//  Created by Vladyslav Shepitko on 27.01.2021.
+//
+
+import PromiseKit
+import Moya
+import SwiftyJSON
+
+struct TokenMappedToTicker: Hashable {
+    let symbol: String
+    let contractAddress: AlphaWallet.Address
+    let server: RPCServer
+
+    init(tokenObject: TokenObject) {
+        symbol = tokenObject.symbol
+        contractAddress = tokenObject.contractAddress
+        server = tokenObject.server
+    }
+
+    init(symbol: String, contractAddress: AlphaWallet.Address, server: RPCServer) {
+        self.symbol = symbol
+        self.contractAddress = contractAddress
+        self.server = server
+    }
+}
+
+protocol CoinTickersResolverType {
+    var tickers: [AddressToPRCServerKey: CoinTicker] { get }
+
+    func coinTickers(enabledObject: [TokenMappedToTicker]) -> Promise<Void>
```

Comment:
Would you rename `enabledObject`? I guess it came from `TokensDataStore`. But not so intention-revealing here.